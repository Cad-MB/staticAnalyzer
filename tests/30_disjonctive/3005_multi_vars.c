{
    int x;
    int y;
    int z;
    x = 1;
    y = 2;
    if (x < y) {
      z = 10;
    } else {
      z = 20;
    }
    if (z >= 15) {
      x = z;
    } else {
      y = z;
    }
    print_all;
  }