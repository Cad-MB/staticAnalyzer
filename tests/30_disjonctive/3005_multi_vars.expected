tests/30_disjunctive/3005_multi_vars.c:15.2-12: [ x in {[1;1] ∨ [20;20]}, y in {[2;2] ∨ [10;10]}, z in {[10;10] ∨ [20;20]} ]
analysis ended